egs/gale_arabic/s5b/local/subword/run_subword.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash -e
+
+# Copyright 2014 QCRI (author: Ahmed Ali)
+#           2019 Dongji Gao
+# Apache 2.0
+
+# This is an example script for subword implementation
+
+num_jobs=120
+num_decode_jobs=40
+decode_gmm=true
+stage=0
+overwrite=false
+num_merge=1000
```

Comment:
what's the meaning of overwrite? It seems to be there to prevent overwriting old files. I'm curious why you would provide an option for this.

num_merges, same as before

Reply:
This is from the previous version. I think it just makes sure that you know what you are doing on data/train/text.